datafusion/expr/src/aggregate_function.rs

```diff
@@ -33,8 +33,6 @@ use strum_macros::EnumIter;
 // https://datafusion.apache.org/contributor-guide/index.html#how-to-add-a-new-aggregate-function
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Hash, EnumIter)]
 pub enum AggregateFunction {
-    /// Count
```

Comment:
🎉 

There are still a bunch of these functions -- at some point we can probably file tickets and spread out the work / fun to migrate them over.

Reply:
I consistently file 1~2 issues for #8708  so the tickets could kept on the first page

Reply:
Perhaps that is how PRs like https://github.com/apache/datafusion/pull/10898 show up -- I can't keep up anymore. Thanks @jayzhan211  -- really nice